tests/foreman/cli/test_errata.py

```diff
@@ -80,113 +74,107 @@
 from robottelo.constants.repos import FAKE_6_YUM_REPO
 from robottelo.constants.repos import FAKE_9_YUM_REPO
 from robottelo.decorators import skip_if_not_set
-from robottelo.test import CLITestCase
 from robottelo.vm import VirtualMachine
 
+
 ERRATUM_MAX_IDS_INFO = 10
 
+CUSTOM_REPO_URL = FAKE_9_YUM_REPO
+CUSTOM_PACKAGE = FAKE_1_CUSTOM_PACKAGE
+CUSTOM_ERRATA_ID = FAKE_2_ERRATA_ID
+CUSTOM_PACKAGE_ERRATA_APPLIED = FAKE_2_CUSTOM_PACKAGE
+
 
+@skip_if_not_set('clients', 'fake_manifest')
+@pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
```

Comment:
These markers are doing the same thing, but in two different ways. Let's consolidate them to one mark/decorator.

@JacobCallahan thoughts on which way we should go, that would lend itself toward future plans for handling test collection/skipping based on available configuration?

Reply:
@mshriver they are not the same, the @skip_if_not_set('clients' was to ensure you had configured the clients section of robottelo properties. As we are moving away from ini file type robottelo properties the point is now moot.

Reply:
Those skip markers still apply, and will continue to apply until the config facade layer is removed - at which point they'll be converted in bulk if a new syntax is necessary. So feel free to continue using them to control test behavior.